src/Alert/Alert.tsx

```diff
@@ -2,23 +2,28 @@ import React, { useState } from "react";
 import { useTranslation } from "react-i18next";
 import styled from "styled-components";
 import { Box } from "../Box";
-import {BoxProps} from "../Box/Box";
+import { BoxProps } from "../Box/Box";
+import { LinkProps } from "../Link/Link";
 import { Icon } from "../Icon";
 import { Link } from "../Link";
 import { Flex } from "../Flex";
 import { Text } from "../Type";
 
 export type AlertProps = BoxProps & {
-    children?: React.ReactNode;
-    className?: string;
-    isCloseable?: boolean;
-    closeAriaLabel?: string;
-    title?: string;
-    type?: "danger" | "informative" | "success" | "warning" | undefined;
-    onClose?: any;
-    controlled?: boolean;
-    style?: React.CSSProperties;
-  };
+  children?: React.ReactNode;
+  className?: string;
+  isCloseable?: boolean;
+  closeAriaLabel?: string;
+  title?: string;
+  type?: "danger" | "informative" | "success" | "warning" | undefined;
+  onClose?: any;
+  controlled?: boolean;
+  style?: React.CSSProperties;
+};
+
+const CollapsedLink = styled(Link)<LinkProps>(() => ({
```

Comment:
Did you add this component because the Link didn't accept a `lineHeight` prop? We should probably add the prop to the Link instead of just adding CSS so it can be leveraged in other cases as well.

Reply:
Good point. I see `typography` is already available on the `Link` component, but the `TypographyProps` is not, that's why I was getting the error when adding it directly on the the `Link`. 

I added `TypographyProps` to the `LinkProps` and removed the styled component.